rpc/src/endpoint/abci_query.rs

```diff
@@ -1,13 +1,12 @@
 //! `/abci_query` endpoint JSONRPC wrapper
 
-use crate::{
-    abci::{Code, Log, Path},
-    block,
-    merkle::proof::Proof,
-    rpc, serializers,
-};
 use serde::{Deserialize, Serialize};
 
+use tendermint::abci::{Code, Log, Path};
+use tendermint::block;
+use tendermint::merkle::proof::Proof;
+use tendermint::serializers;
```

Comment:
Just out of curiosity: are all these import changes due to a different formatter? Is there a re reason why all the `use crate:: ...`s are replaced in that fashion?

Reply:
Personal touch really, optimised for editability and tracebility of dependencies. I usually one liner per top-level module in a crate.

Reply:
Ok for me. Ideally, these should be consistent in all our crates though.